cli/cmd/bash.go

```diff
@@ -28,6 +28,7 @@ Auto find the bash of the current env as the capture target.`,
 func init() {
 	bashCmd.PersistentFlags().StringVar(&bc.Bashpath, "bash", "", "$SHELL file path, eg: /bin/bash , will automatically find it from $ENV default.")
 	bashCmd.PersistentFlags().StringVar(&bc.Readline, "readlineso", "", "readline.so file path, will automatically find it from $BASH_PATH default.")
+	bashCmd.Flags().IntVarP(&bc.ErrNo, "ErrNo", "e", 128, "only show the command which return the Error Number")
```

Comment:
`Errno` arguments must use lowercase letters like `bash` \ `readlineso`.

Reply:
copy that